tokens/pickle.json

```diff
@@ -0,0 +1,8 @@
+{
+    "ethereum_address": "0x429881672B9AE42b8EbA0E26cD9C73711b891Ca5",
+    "name": "PickleToken",
+    "symbol": "PICKLE",
+    "decimals": 18,
+    "reference": "",
+    "reference_hash": ""
+}
```

Comment:
Please add "icon" field (even if it's empty if you can't convert it) and "aurora_address" - we deployed a contract for you on `0x291c8fceaca3342b29cc36171deb98106f712c66`